lib/process-services-cloud/src/lib/task/task-header/task-header-cloud.module.ts

```diff
@@ -21,6 +21,8 @@ import { MaterialModule } from '../../material.module';
 import { DataTableModule, TemplateModule, CardViewModule, CoreModule } from '@alfresco/adf-core';
 import { TaskHeaderCloudComponent } from './components/task-header-cloud.component';
 import { TaskHeaderCloudService } from './services/task-header-cloud.service';
+import { CompleteTaskDirective } from './directives/complete-task.directive';
```

Comment:
I don't think this directive should be imported by the task header module
We should have a task.module